projects/experimental/components/compass/compass.style.less

```diff
@@ -0,0 +1,39 @@
+:host {
+    --t-degrees: 0;
+    position: relative;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    color: var(--tui-primary);
+    border-radius: 50%;
+    box-sizing: border-box;
+    transform-style: preserve-3d;
+    width: 1rem;
+    height: 1rem;
+    border: 0.1875rem solid var(--tui-base-02);
+    box-shadow: 0 0 0.4375rem rgba(0, 0, 0, 0.3);
+    transform: rotate(var(--t-degrees));
+
+    &:before {
+        content: '';
+        position: absolute;
+        top: 0;
+        left: 0;
+        height: 100%;
+        width: 100%;
+        border-radius: inherit;
+        background: currentColor linear-gradient(-45deg, rgba(0, 0, 0, 0.2), transparent);
+        transform: rotate(calc(-1 * var(--t-degrees)));
```

Comment:
Why not just rotate the beak?

Reply:
It doesn't work for me for the beak

https://github.com/taiga-family/taiga-ui/assets/12021443/cdc32c33-7ed8-45d2-a5de-d490bb267918

Reply:
To rotate just beak it has to be centered properly. Check out my very last comment:
https://github.com/taiga-family/taiga-ui/pull/5439#issuecomment-1731551049
The numbers I gave there are for the case when border is moved to the after element, so you would need to subtract border width from them, I guess.

Reply:
fixed